rna/models.py

```diff
@@ -43,7 +55,7 @@ class Release(TimeStampedModel):
     is_public = models.BooleanField(default=False)
     bug_list = models.TextField(blank=True)
     bug_search_url = models.CharField(max_length=2000, blank=True)
-    system_requirements = models.TextField(blank=True)
+    system_requirements = models.ForeignKey(SystemRequirements, on_delete=models.CASCADE)
```

Comment:
Thanks for the correction. However, I'm not sure how we can implement this. So we want to add an optional foreign key field to the system_requirement table which gets activated only when the field of **system_requirements** in the **release** table is empty. How do I implement this?

What I've done is this:

- Create a foreign key field say **system_requirement_id** in the release table alongside the **system_requirement** text field.
- Now we can set the **system_requirement_id** field to be **blank=True, null=True**.

But, now I'm not able to understand how to implement this logic that the user needs to set system_requirement_id only when the system_requirement field is blank?